Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -2889,11 +2895,32 @@ void DrawMol::bondNonRing(const Bond &bond, double offset, Point2D &l2s,
   const Atom *thirdAtom = nullptr;
   const Atom *fourthAtom = nullptr;
 
+  bool begTrunc(!atomLabels_[begAt->getIdx()]);
+  bool endTrunc(!atomLabels_[endAt->getIdx()]);
```

Comment:
we normally initialize with assignment (and I find it more readable):
```suggestion
  bool begTrunc = !atomLabels_[begAt->getIdx()];
  bool endTrunc = !atomLabels_[endAt->getIdx()];
```

Reply:
Yep.  But bool begTrunc = atomLabels_... is an error, and so I changed it to the above and then realised I needed the !.